tests/test_sanitize.py

```diff
@@ -39,9 +39,6 @@ def test_sanitize():
                 ('no-soma.asc',
                  '{} has an invalid or no soma'.format(Path(PATH, 'no-soma.asc'))),
 
-                ('negative-diameters.asc',
-                 '{} has negative diameters'.format(Path(PATH, 'negative-diameters.asc'))),
-
```

Comment:
Can you replace this by a test of the new diameters

Reply:
done!